src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(x) {
+    count++;
+
+    return function(y) {
```

Comment:
its ok, but u can rename `x y` for more understandable names)

Reply:
I agree with Dima - you should use better naming